codegen/src/Azure.Iot.Operations.ProtocolCompiler/T4/communication/rust/Command/t4/RustCommandInvoker.tt

```diff
@@ -164,6 +164,8 @@ where
             .request_topic_pattern(REQUEST_TOPIC_PATTERN)
             .command_name("<#=this.commandName.AsGiven#>")
             .topic_token_map(topic_token_map)
+            .response_topic_prefix(options.response_topic_prefix.clone())
```

Comment:
I don't think these two lines should need the clone since you aren't using these fields again! (If it complains because you're still using options, then ignore this comment)

Reply:
The compiler complained.  I didn't originally have the clone, but the build failed without it.